fml/raster_thread_merger_unittests.cc

```diff
@@ -616,11 +487,148 @@ TEST(RasterThreadMerger, SetMergeUnmergeCallback) {
 
   raster_thread_merger->DecrementLease();
   ASSERT_EQ(2, callbacks);
+}
 
-  term1.Signal();
-  term2.Signal();
-  thread1.join();
-  thread2.join();
+TEST(RasterThreadMerger, MultipleMergersCanMergeSameThreadPair) {
+  TaskQueueWrapper queue1;
+  TaskQueueWrapper queue2;
+  fml::TaskQueueId qid1 = queue1.GetTaskQueueId();
+  fml::TaskQueueId qid2 = queue2.GetTaskQueueId();
+  const auto raster_thread_merger1_ =
```

Comment:
s/raster_thread_merger1_/raster_thread_merger1

That naming scheme is reserved for instance variables.